src/PluralResolver.js

```diff
@@ -3,74 +3,260 @@ import baseLogger from './logger.js';
 // definition http://translate.sourceforge.net/wiki/l10n/pluralforms
 /* eslint-disable */
 let sets = [
-  { lngs: ['ach','ak','am','arn','br','fil','gun','ln','mfe','mg','mi','oc', 'pt', 'pt-BR',
```

Comment:
can we exclude those rules stuff from getting changed by prettier (it does not really add to readablity)

Reply:
yes, we can ignore specific files - I'll add that